geoviews/operation/projection.py

```diff
@@ -386,6 +387,11 @@ class project_image(_project_operation):
         when applying project_image, backends that support linked streams
         update RangeXY streams on the inputs of the operation.""")
 
+    mask_extrapolated = param.Boolean(default=False, doc="""
```

Comment:
```suggestion
    mask_extrapolated = param.Boolean(default=True, doc="""
```

We want this on by default, so hvplot can get the benefit.